self-tests.yml

```diff
@@ -686,7 +685,7 @@ buildvariants:
       goos: linux
       GOROOT: /opt/golang/go1.16
       RUN_EC2_SPECIFIC_TESTS: true
-      mongodb_url: https://fastdl.mongodb.org/linux/mongodb-linux-aarch64-ubuntu1804-4.2.18.tgz
+      mongodb_url: https://fastdl.mongodb.org/linux/mongodb-linux-aarch64-ubuntu1804-4.4.18.tgz
```

Comment:
Left this distro alone on the assumption it's not here to test the app servers

Reply:
Is this a reason to continue testing on 18.04?

Reply:
I mean, the agent runs on both ubuntu1804 and ubuntu2004.
I think the idea of this variant is just to make sure we test the ARM architecture.